test/e2e/indexTable/indexTable.spec.ts

```diff
@@ -5,6 +5,8 @@ test('IndexTable', async ({ page }) => {
   const indexTablePage = new IndexTablePage(page);
   await indexTablePage.goto();
 
+  await indexTablePage.tableRow.waitFor();
```

Comment:
It's being flaky, sometime the table is not loaded when it gets captured. I added this line to make sure that the table has been loaded by looking at tableRow sub component